entries/d/didiglobal.com.json

```
@@ -0,0 +1,31 @@
+{
+  "Didi": {
+    "domain": "didiglobal.com",
+    "tfa": [
+      "sms"
+    ],
+    "notes": "Didi decides when 2FA is required; user cannot enable 2FA for every login.",
+    "categories": [
+      "transport"
+    ],
+    "regions": [
+      "cn",
+      "mx",
+      "au",
+      "jp",
+      "br",
+      "co",
+      "hk",
+      "tw",
+      "cl",
+      "cr",
+      "pa",
+      "ru",
+      "pe",
+      "ec",
+      "do",
+      "eg",
+      "ar"
```

```suggestion
      "ar",
      "au",
      "br",
  	  "cl",
  	  "cn",
  	  "co",
  	  "cr",
  	  "do",
  	  "ec",
  	  "eg",
  	  "hk",
  	  "jp",
  	  "mx",
  	  "nz",
  	  "pa",
  	  "pe",
  	  "tw"
```

Here's an alphabetized list of regions